docs/api-testing/composer.md

```diff
@@ -49,31 +49,31 @@ To edit a test at any time, on the **Projects** page, on the **Tests** tab, hove
 <img src={useBaseUrl('/img/api-testing/edit-test-nav.png')} alt="Navigating to the test editor" width="300"/>
 
 ### Add Test Components
-When test components are combined, they act as our test logic. There are three component types available in API Testing:
+When test components are combined, they act as our test logic. See the following pages for more information about the components types available in API Testing:
   * [I/O Request Test Components](/api-testing/composer/io-components)
   * [Assertion Test Components](/api-testing/composer/assertion-components/)
   * [Logical Test Components](/api-testing/composer/logical-components/)
+  * [Other Components](/api-testing/composer/other-components/)
+
 
 #### Add an I/O Request Test Component
 To create a simple `GET` request and validate that response is correct:
-  1. In API Testing, on the **Compose** page, click the **Add component** button.
+  1. In API Testing, on the **Compose** page, click **Add child component**.
 
-  <img src={useBaseUrl('/img/api-testing/add-component-nav.png')} alt="Navigating to the Add component screen" width="600"/>
+  <img src={useBaseUrl('/img/api-testing/add-component-nav.png')} alt="Navigating to the Add component screen"/>
```

Comment:
As per our [style guide](https://wiki.saucelabs.com/pages/viewpage.action?spaceKey=PM&title=Technical+Documentation+Style+Guide#TechnicalDocumentationStyleGuide-Images,Screenshots,andVideos) our usage of images and screenshots should be minimal.

Reply:
will address in a future content update